ScribusGeneratorBackend.py

```diff
@@ -417,6 +419,14 @@ def generate_templates(self, root, data: list) -> list:
 
         # Clean & write single SLA file (merge-mode only)
         if merge_mode:
+            output_element = ET.fromstring(output)
```

Comment:
i dont quite understand the purpose of these next 4lines. What problem do you fix here ?

Reply:
i.e. 422 to 425.

426 is a nice addition I forgot, thanks.

Reply:
The reason of this 4-lines addition was to fix the issue missing following tags.

`<SCRIBUSUTF8NEW Version="1.5.7">.... </SCRIBUSUTF8NEW>`

Sorry, right now, I am not sure whether this 4-lines effective. This issue was not appeared when I tried again with commented this codes out.